jenkins-builds/7.0.UP00-19.03-mc

```diff
@@ -24,7 +24,9 @@
  ipyparallel-6.2.3-CrayGNU-19.03-python3.eb
  jupyter-1.0.0-CrayGNU-19.03.eb
  LAMMPS-22Aug2018-CrayGNU-19.03.eb                  --set-default-module
+ LLVM-8.0.0-CrayGNU-19.03.eb
```

Comment:
Shouldn't we include this in the GPU list?

Reply:
It's not present in the current GPU list.

Reply:
i am preparing a recipe for clang, it will replace llvm.

Reply:
I'm removing version `8.0.0` from the list, after discussing with @jfavre: the updated recipe will still be in the repository and it will be available for next releases of `Mesa` and `ParaView`, if applicable.